### PR TITLE
Add od recipe

### PR DESCRIPTION
**Pull request by proinsias**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by proinsias** [comment]
I am willing to be listed in the maintainer section.

**Comment by proinsias** [comment]
@conda-forge/help-python – I believe this is ready for review now. Not sure why AppVeyor builds are failing for these PRs.

File: recipes/od/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: d1c94cd3af5fdb13ca27c4d3e1601c77028f36bbba9e78c324daf0c1ac06e3ab
+
+build:
+  number: 0
```
Comment (marcelotrevisani) [inline recipes/od/meta.yaml]:
```suggestion
  noarch: python
  number: 0
```

File: recipes/od/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  imports:
+    - od
+
+about:
+  home: "https://github.com/epsy/od"
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
+  summary: "Shorthand syntax for building OrderedDicts"
+  doc_url:
```
Comment (marcelotrevisani) [inline recipes/od/meta.yaml]:
```suggestion
```

**Comment by proinsias** [comment]
@marcelotrevisani – I've updated as per your suggestions.

**Pull request merged**